### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.resourcegraph.models.error_response_py3.ErrorResponse
  - azure.mgmt.resourcegraph.models.error_response_py3.ErrorResponseException
  fullName: azure.mgmt.resourcegraph.models.error_response_py3
  langs:
  - python
  module: azure.mgmt.resourcegraph.models.error_response_py3
  name: error_response_py3
  source:
    id: error_response_py3
    path: azure-mgmt-resourcegraph\azure\mgmt\resourcegraph\models\error_response_py3.py
    remote:
      branch: master
      path: azure-mgmt-resourcegraph\azure\mgmt\resourcegraph\models\error_response_py3.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.resourcegraph.models.error_response_py3
references:
- fullName: azure.mgmt.resourcegraph.models.error_response_py3.ErrorResponse
  isExternal: false
  name: ErrorResponse
  parent: azure.mgmt.resourcegraph.models.error_response_py3
  uid: azure.mgmt.resourcegraph.models.error_response_py3.ErrorResponse
- fullName: azure.mgmt.resourcegraph.models.error_response_py3.ErrorResponseException
  isExternal: false
  name: ErrorResponseException
  parent: azure.mgmt.resourcegraph.models.error_response_py3
  uid: azure.mgmt.resourcegraph.models.error_response_py3.ErrorResponseException
